README.md

```diff
@@ -31,7 +33,17 @@ Run the following commands:
 
 ### Backend
 
-To help with backend implementation, you can reference the `count` demo feature. 
+To help with backend implementation, you can reference the `count` demo feature.
+
+#### Authentication - JWT
+
+Ensure that you have these packages installed: `pip3 install pyjwt` to ensure your project works well with the authorization system. Navigate to your `backend` folder and create yourself a `.env.development`. You will need to run the following command to generate yourself a secret key: `openssl rand -hex 32`. To confirm that the route works as expected, clear your database and run the test data generated for the `teams.py` test data.
+
+The format for the `.env.development` file is:
+
+SECRET_KEY=1e2b4e4140036f4ed0131bd6d3a2061181735211c2364ccf47ac410a98c03682

```

Comment:
Maybe take out the pre-generated key here so we're not tempted to just re-use this one.

Reply:
Done